git_cc/update.py

```diff
@@ -1,11 +1,17 @@
 """Update the git repository with Clearcase manually, ignoring history"""
 
-from common import *
-import sync, reset
+from __future__ import print_function
+
+from .common import *
+from . import reset
+from . import sync
+
 
 def main(message):
```

Comment:
At the moment I am just using update. I agree, it is not ideal :) You can even lose data so you have to be careful when doing the update. However, update allowed us to get up to speed quickly to see whether a git bridge would be an option for us.

Using rebase would be better and will probably be the way forward for us after our initial usage of git-cc. Unfortunately, the initial rebase of our ClearCase repos aborts. I will have to look into that.